File: Python/p1.py
r, c = map(int, input("Enter rows and columns: ").split())
a = []
print("Enter elements:")
for i in range(r):
    row = list(map(int, input().split()))
    a.append(row)

print("Rotated matrix:")
for j in range(c):
    for i in range(r-1, -1, -1):
        print(a[i][j], end=' ')
    print()

File: C/p1.c
#include <stdio.h>

int main() {
    int r, c;
    printf("Enter rows and columns: ");
    scanf("%d %d", &r, &c);

    int a[r][c];

    printf("Enter elements:\n");
    for(int i = 0; i < r; i++) {
        for(int j = 0; j < c; j++) {
            scanf("%d", &a[i][j]);
        }
    }

    printf("Rotated matrix:\n");
    for(int j = 0; j < c; j++) {
        for(int i = r - 1; i >= 0; i--) {
            printf("%d ", a[i][j]);
        }
        printf("\n");
    }

    return 0;
}
